main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for nickname, player in data.items():
+
+        race_db, _ = Race.objects.get_or_create(
+            name=player["race"]["name"],
+            description=player["race"]["description"]
+        )
+
+        if player["guild"]:
+            guild_db, _ = Guild.objects.get_or_create(
+                name=player["guild"]["name"],
+                description=player["guild"]["description"]
+            )
+        else:
+            guild_db = None
```

Comment:
You can simplify your code
```suggestion
        guild_db = None
        if player["guild"]:
            guild_db, _ = Guild.objects.get_or_create(
                name=player["guild"]["name"],
                description=player["guild"]["description"]
            )
```